docs/contributor_guide/index.rst

```diff
@@ -32,15 +32,16 @@ All are welcome!
 To report bugs, request features, or ask usage questions please use the
 `appropriate channels as outlined on GitHub <https://github.com/fairlearn/fairlearn/issues/new/choose>`_.
 
-.. _developer_calls:
+.. _community_calls:
 
-Developer calls
+Community calls
 ---------------
 
-We have a monthly contributor meeting every second Thursday of the month at
-11am New York City time. Check what this means in your time zone with
+We have a weekly community call every Thursday at 11am Eastern time.
```

Comment:
```suggestion
We have a weekly community call every Thursday at 11am Eastern US time (ET).
```
or north american. I find it odd that that timezone doesn't have an indication that it's relative to the US basically, and I was very confused the first time I saw it.

Reply:
It's actually called Eastern Time, but I can make it more specific. https://www.timeanddate.com/time/zones/et